svcomp/src/main/java/com/dat3m/svcomp/SVCOMPRunner.java

```diff
@@ -59,6 +59,8 @@ private void property(String p) {
 
     public static void main(String[] args) throws Exception {
 
+        String final boundsFilePath = System.getenv("DAT3M_OUTPUT") + "/bounds.csv";
+
```

Comment:
Why declare it here if you use it only at lines 79+? Move it down to its first usage.
Also, put the "final" first :)